### ISSUE TITLE
Multiple cuts fail in the drawing API

### ISSUE THREAD

**Issue by paulftw** [sgenoud/replicad#50]
Below code produces solid disk, but commenting out either `.cut` produces a disk with a hole.
```
function main({drawCircle}) {
 return drawCircle(10)
   .cut(drawCircle(2).translate(5, 5))
   .cut(drawCircle(1))
   .sketchOnPlane('XZ')
   .extrude(5)
}
```
Visualizer draws edges where openings are supposed to be, but they appear to be filled.

A [bigger example](https://scriptcad.com/paulftw/cadNotCut):

<img width="403" alt="Screen Shot 2022-07-14 at 5 50 00 PM" src="https://user-images.githubusercontent.com/769142/179011387-a61105ec-64ed-463c-98fa-d45dbc93b549.png">

```
  return [
    drawCircle(10),
    drawCircle(10).cut(drawCircle(2).translate(5, 5)),
    drawCircle(10).cut(drawCircle(2)).cut(drawCircle(2).translate(5, 5)),
  ].map((x, i) =>
    x
      .sketchOnPlane('XZ')
      .extrude(5)
      .translate(i * 25, 0, 25),
  )
```

**Comment by sgenoud** [comment]
Thanks, I found a solution, it is in a branch I will merge once I found a solution to the more complicated bug related to triangle fusion!

**Comment by sgenoud** [comment]
It should be fixed in 0.12.4 (and on the visualizer) - do you confirm?

**Comment by paulftw** [comment]
Tested in 0.12.5 (latest as of today). Works as expected now!
